a10_octavia/controller/worker/flows/vthunder_flows.py

```diff
@@ -229,6 +243,33 @@ def _get_vthunder_for_amphora_subflow(self, prefix, role):
             requires=constants.LOADBALANCER,
             inject={"role": role},
             provides=constants.COMPUTE_ID))
+
+        # if no compute, use spare vThunder and prepare network for spare vThunder
```

Comment:
For MapLoadbalancerToAmphora,
- It returns a vthunder.id. which may be a 
1. vthunder.id for this project. 
if this project already have a vthunder for it. per line 228:
```
        vthunder = self.vthunder_repo.get_vthunder_by_project_id(
            db_apis.get_session(),
            loadbalancer.project_id)
```
2. Or a spare vthunder.id.
which we have a spare vthunder with only has interfaces for amp_boot_network_list 
```
            vthunder = self.vthunder_repo.get_spare_vthunder(
                db_apis.get_session())
```

- or it returns NULL. (if no vThunder for this project  and no spare vthunder)


Therefore, for project vthunder or spare vthunder case. They both gose to _get_vthunder_for_amphora_subflow().
And in previous code, didn't find any place we prepare interfaces for spare vthunder. 
Also tried create loadbalancer based on spare vthunder and it can't work. So, I assume we never implement this before.
So, add these part.